internal_workflow_testsuite.go

```diff
@@ -277,12 +277,10 @@ func (env *testWorkflowEnvironmentImpl) executeWorkflow(workflowFn interface{},
 	case reflect.String:
 		workflowType = workflowFn.(string)
 	case reflect.Func:
-		// auto register workflow if it is not already registered
```

Comment:
This makes it test code simpler, i was hoping we make it easier not to register explicitly for test code.